src/main/drivers/timer_common.c

```diff
@@ -158,5 +158,19 @@ ioTag_t timerioTagGetByUsage(timerUsageFlag_e usageFlag, uint8_t index)
 #endif
     return IO_TAG_NONE;
 }
+
+const timerHardware_t *timerGetByUsage(timerUsageFlag_e usageFlag, uint8_t index)
+{
+    uint8_t currentIndex = 0;
```

Comment:
This will have to be hardcoded to return `NULL` if `USE_UNIFIED_TARGET` is not defined.

Reply:
Also thrown out now.